content/src/fieldConfig/config.json

```diff
@@ -302,7 +302,7 @@
     "officeLink": "https://innovation.nj.gov/",
     "linkOneText": "Give Us Feedback",
     "gitHubLink": "https://github.com/newjersey",
-    "officeTextOne": "Made with ",
+    "madeWithText": "Made with ",
```

Comment:
I love that you've rethought the naming of the variables here! Great use of campsite rules.

[dust] Given that the Office of Innovation is no longer its own link, do you think it makes sense to keep "by the" and "Office of Innovation" as separate items or do you think we should consider combining them? If we don't use `officeOfInnovationText` elsewhere, does it still make sense to have these as separate config items?

Reply:
This works if you do use the non-breaking spaces in the string approach.

I think its highly unlikely, but someone could edit one of the items and remove the non-breaking spaces. The current solution would account for that, but this proposal is simpler. I'm not really torn one way or another.